src/display/xfa_layer.js

```diff
@@ -13,9 +13,23 @@
  * limitations under the License.
  */
 
+/** @typedef {import("./display_utils").PageViewport} PageViewport */
+/** @typedef {import("../../web/interfaces").IPDFLinkService} IPDFLinkService */
+
 import { warn } from "../shared/util.js";
 import { XfaText } from "./xfa_text.js";
 
+/**
+ * @typedef {Object} XfaLayerParameters
+ * @property {PageViewport} viewport
+ * @property {HTMLDivElement} div
+ * @property {Object} xfaHtml
+ * @property {PDFPageProxy} [pdfPage]
```

Comment:
I don't see `pdfPage` being used anywhere in this file aside from this comment; is that correct?

Reply:
Good catch, that one looks completely unused here!
I'll remove this from the typedef, and also go through the code and make sure that we stop passing in an unused parameter to the `XfaLayer` (since it's just confusing as-is).